src/libraries/pkg/Microsoft.NETCore.Platforms/runtime.compatibility.json

```diff
@@ -1061,6 +1061,124 @@
     "any",
     "base"
   ],
+  "ios": [
+    "ios",
+    "unix",
+    "any",
+    "base"
+  ],
+  "ios-arm64": [
```

Comment:
I think we should add iOS Simulator and Device as separate RIDs extending from the generic iOS RID.

Reply:
that would mean the rid would need to be called something like `iosdevice-arm64` and I don't think that's a good idea.

Reply:
Apple decided to split them up into two separate SDKs/targets so I don't think we have a choice.

Reply:
`ios-arm64` - device RID
`ios-x64` - simulator RID

Both have parent RID `ios`. As I wrote above explicitly stating `device` in the RID is something I think we should try to avoid

Reply:
That will break down as soon as Apple introduces the rumored ARM MacBook. I really don't think we should bake in the assumption about which architectures are a device runtime vs. simulator runtime.

Reply:
why? such device would use different RID called `osx-arm64`

Reply:
I'm talking about the iOS Simulator running on an ARM MacBook. We'd need to use a simulator build of the runtime there, not a device one.

Reply:
Well, we don't know if simulator would be needed or not but even in that case the existing naming convention covers that with `ios-arm64-jit` like format.

Reply:
We discussed offline and I wasn't aware of the "additional qualifiers" specified in https://github.com/dotnet/runtime/blob/master/src/libraries/pkg/Microsoft.NETCore.Platforms/readme.md#naming-convention
> [additional qualifiers] can be things like aot. Used to further differentiate different platforms.

This settles my concern.